helmfile.d/helmfile-00.uninstall.yaml

```diff
@@ -0,0 +1,17 @@
+bases:
+  - snippets/defaults.yaml
+---
+bases:
+  - snippets/env.gotmpl
+---
+bases:
+  - snippets/derived.gotmpl
+---
+{{ readFile "snippets/templates.gotmpl" }}
+
+releases:
+  - name: gatekeeper-operator
+    installed: false
+    namespace: default
+    chart: not-used-but-required
+    # TODO: remove definition of this release in release 0.16.3
```

Comment:
I suggest keeping uninstalls around until the next minor is released. And that is also very quick. We have to start getting used to a deprection process.

Reply:
yes, that is the intent